src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Create.cs

```diff
@@ -219,17 +221,16 @@ private static bool IsGlobalSuppressMessageAttribute(ISyntaxFactsService syntaxF
             syntaxFacts.StringComparer.Equals(identifierName, nameof(SuppressMessageAttribute));
     }
 
-    private static void TryAddGlobalAliasInfo(
+    private static void TryAddAliasInfo(
         ISyntaxFactsService syntaxFacts,
+        ref HashSet<(string alias, string name, int arity)>? aliasInfo,
         ref HashSet<(string alias, string name, int arity)>? globalAliasInfo,
         SyntaxNode node)
     {
         if (!syntaxFacts.IsUsingAliasDirective(node))
             return;
 
         syntaxFacts.GetPartsOfUsingAliasDirective(node, out var globalToken, out var alias, out var usingTarget);
-        if (globalToken.IsMissing)
-            return;
```

Comment:
this was the bug.  `IsMissing` is for when the parser thinks a token must be there but doesn't see it.  e.g. `if x == y)` will have a 'missing' open paren token.  but if you just have `using x = u;` there is no missing 'global' token there as that's just a legal using statement with a *default* global token.

this bug was making it so that *every* alias in the project was being treated as a global alias.   We'd still only look for that alias if the name it bound to was hte name of hte symbol we were looking for.  but this was still unnecessary work.